src/track/timesignature.h

```diff
@@ -0,0 +1,49 @@
+// signature.h
+// Created on: 06/11/2019 by Javier Vilarroig
+
+#pragma once
+
+namespace mixxx {
+
+// Musical Time signature
+// Right now is to be used for bar detection only, so only the beats per bar is useful
+class TimeSignature final {
+  private:
+    int m_beatsPerBar;
+    int m_noteValue;
+
+  public:
+    inline TimeSignature(int beatsPerBar, int noteValue)
```

Comment:
All methods that are defined inside the class body are implicitly inlined. Please remove the keyword.

Reply:
Done

Reply:
I still see inline here. Please commit the corresponding changes timely when marking notes as done. Otherwise I have to review everything twice.